server/api/models/user.js

```diff
@@ -17,6 +17,10 @@ module.exports = {
       type: "string",
```

Comment:
models should start with capital letter (eg. User.js) so that we avoid confusion (all our ORM calls use capital letters)